pd/src/state.rs

```diff
@@ -96,19 +98,29 @@ INSERT INTO blobs (id, data) VALUES ('gc', $1)
                     identity_key,
                     consensus_key,
                     sequence_number,
-                    validator_data,
-                    voting_power
-                ) VALUES ($1, $2, $3, $4, $5)",
+                    name,
+                    website,
+                    description,
+                    voting_power,
+                    validator_state,
+                    unbonding_epoch
+                ) VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9)",
                 validator.identity_key.encode_to_vec(),
                 validator.consensus_key.to_bytes(),
                 validator.sequence_number as i64,
-                validator.encode_to_vec(),
+                validator.name.clone(),
+                validator.website.clone(),
+                validator.description.clone(),
```

Comment:
You don't need to clone any of these, you can refer to them by reference and it will work just fine.